jenkins_job_manager/raw_ext.py

```diff
@@ -0,0 +1,81 @@
+"""
+Contains extensions for raw xml loading with jjb
+"""
+import os
+import logging
+import jinja2
+import xml.etree.ElementTree
+import jenkins_jobs.modules.base
+from jenkins_jobs.xml_config import XmlJob, XmlJobGenerator
+
+log = logging.getLogger("jjm")
+
+
+def xml_escape(s):
+    """hacks ElementTree to give us strings escaped for xml"""
+    ET = xml.etree.ElementTree
+    a = ET.Element("a")
+    a.text = s
+    # spit out element as string, slice out the element text
+    escaped_s = ET.tostring(a, encoding="unicode")[3:-4]
```

Comment:
It might be trying to be too clever (I'm guessing `<a>` and `</a>` here, but can't the tag be longer via attributes?

Reply:
Yeah, I had to double take on my slice syntax. might wanna say why these specific indices